integration-tests-cypress/support/index.ts

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Please run prettier on all the files and make sure there a no leading empty lines and all files end in a newline. There is prettier config in the repo.

As an aside, we should make sure that we're checking that in CI, but can be a separate PR.